src/components/ui/AppColorPicker.vue

```diff
@@ -262,6 +279,63 @@ export default class AppColorPicker extends Vue {
   debouncedChange (channel: string, color: IroColor) {
     this.$emit('change', { channel, color })
   }
+
+  startMouseDrag (event: MouseEvent) {
+    this.dragging = true
+    this.lastPointerPosition = { x: event.clientX, y: event.clientY }
+  }
+
+  stopMouseDrag () {
+    this.dragging = false
+  }
+
+  startTouchDrag (event: TouchEvent) {
+    this.lastPointerPosition = { x: event.touches[0].clientX, y: event.touches[0].clientY }
+  }
+
+  relativeMove (newPosition: PointerPosition) {
+    if (!this.$refs.card) {
```

Comment:
Up to you on this one, but I prefer to use refs like here: https://github.com/fluidd-core/fluidd/blob/4cf6f4348931d86460b5595951466e08867c81de/src/components/ui/AppChart.vue#L39-L40

Reply:
Oh yeah, I like consistency. Will change that.

Reply:
We got both approaches all around the app, so consistency is long gone! 😁